pycytominer/cyto_utils/cells.py

```diff
@@ -779,12 +785,20 @@ def merge_single_cells(
 
             sc_df = normalize(profiles=sc_df, **normalize_args)
 
+        # In case platemap metadata is provided, use pycytominer.annotate for metadata
+        if platemap is not None:
+            sc_df = annotate(
+                profiles=sc_df, platemap=platemap, output_file="none", **kwargs
```

Comment:
we've historically stayed away from `kwargs` - not for any particular reason except to maintain more explicit function arguments.

Besides decreasing optional function arguments, is there a good reason to introduce `kwargs` back? If so, then we probably should name the argument `annotation_kwargs`.

Reply:
The conversation within the PR led me to include `output` and `annotate` as part of `merge_single_cells`, though I may have misunderstood the intent. Using `kwargs` here allowed for some flexibility and somewhat sidesteps the notion of having "too many parameters" for a method or function (specifically avoiding having to rewrite them within the function where they're otherwise already outlined). This is more of a general code sensibility topic than a strict standard, and `*args/**kwargs` blurs this line a bit I feel.

We could rename here to `annotation_kwargs` as you suggested. I can understand `kwargs` being a difficulty too, we could lean on the developer to create their own merge -> annotate -> output flow within their code, calling each function separately. Thoughts?

Also wanted to include a reference on function parameter count from Python Code Quality Authority (PyCQA):
- [Pylint R0913](https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/too-many-arguments.html): "Used when a function or method takes too many arguments." This is only linting, so it's up to each community to decide what's ultimately best in terms of approach.

Reply:
Just another quick note here: in looking at this further, I noticed that `output.output_type` is why we utilize `**kwargs` for both annotation and output within `merge_single_cells` (as well as in those respective functions). It may be best to leave this as a generalized name so as to not confuse.

Reply:
I am in favor of reducing confusion. @shntnu mentions many alpha testers available. I'm in favor merging this in as you feel reduces most confusion, then receiving feedback from these testers (including Erik) 

> we could lean on the developer to create their own merge -> annotate -> output flow within their code, calling each function separately.

I am generally in favor of leaning on the developer, but I also lean towards making a developer's life easier. See comment above about alpha testers. If they deem kwargs too confusing (or if they end up establishing their own flow we can drop kwargs in the future.

Reply:
Sounds good - I'd also love to hear back from the testers on preference here, deferring to those who make most use of it.

Additional food for thought (and may not apply or be debatable): ["Preference for Deep Classes" by John Ousterhout](https://youtu.be/bmSAYlu0NcY?t=753).